main_pullrequest.go

```diff
@@ -117,7 +117,7 @@ func processGitHubPullRequest(
 				postGitHubMessage(ctx, pr, log, msg)
 			}
 		}
-
+	case "edited":
```

Comment:
You need a fallthrough here too, otherwise it won't run for the above actions. Or call it twice.

Reply:
Sorry, this was wrong, the lone "edited" fix is the only thing we need. Pre-approving, you know what to do!